docs/src/snap/reference/annotations.md

```diff
@@ -24,6 +24,7 @@ the bootstrap configuration.
 | `k8sd/v1alpha1/calico/autodetection-v6/skipInterface` | Enable IP auto-detection based on interfaces that do not match the given regex. | string |
 | `k8sd/v1alpha1/calico/autodetection-v6/canReach` | Enable IP auto-detection based on which source address on the node is used to reach the specified IP or domain. | string |
 | `k8sd/v1alpha1/calico/autodetection-v6/cidrs` | Enable IP auto-detection based on which addresses on the nodes are within one of the provided CIDRs. | []string (comma separated) |
+| `k8sd/v1alpha1/cilium/cni-exclusive` | Make Cilium take ownership over the `/etc/cni/net.d` directory on the node, renaming all non-Cilium CNI configurations to `*.cilium_bak`. This ensures no Pods can be scheduled using other CNI plugins during Cilium agent downtime. Set this to "false" if you wish to use other CNIs such as Multus. | "true"\|"false" |
```

Comment:
nit: this sounds like you need to set one of the values.
`false`/not set is the default. Please rephrase to something like
```
Set to "true" if you do not want to allow multiple CNIs.
```

Reply:
Yes, the initial implementation expects one of these values. I'll update it.